app/templates/application.hbs

```diff
@@ -4,7 +4,6 @@
         {{#link-to 'ladder'}}Ladder{{/link-to}}
         {{#link-to 'rules'}}Rules{{/link-to}}
         {{#link-to 'history'}}History{{/link-to}}
-        {{#link-to 'admin'}}Admin{{/link-to}}
```

Comment:
you can switch between pages as normal now, when admin page loads, the init function checks firebase if the user is already authenticated.